ambry-api/src/main/java/com/github/ambry/store/StoreStats.java

```diff
@@ -50,11 +51,12 @@
    * value is {@link StatsSnapshot}.
    * @param statsReportTypes the specified {@link StatsReportType} to fetch
    * @param referenceTimeInMs the reference time in ms until which deletes and expiration are relevant
+   * @param accountIdToExclude the list of account id to exclude from the {@link StatsSnapshot}.
    * @return a map of {@link StatsReportType} to {@link StatsSnapshot}
    * @throws StoreException
    */
-  Map<StatsReportType, StatsSnapshot> getStatsSnapshots(Set<StatsReportType> statsReportTypes, long referenceTimeInMs)
-      throws StoreException;
+  Map<StatsReportType, StatsSnapshot> getStatsSnapshots(Set<StatsReportType> statsReportTypes, long referenceTimeInMs,
+      List<Short> accountIdToExclude) throws StoreException;
```

Comment:
Might be best to avoid new uses of short for account/container ids since they will change soon.  Perhaps Numeric, or round up to Integer?

Reply:
Just talked to Ze, since the account and container ids conversion is not requested by our customers, we are not going to do it any more. I am think that we should probably keep use shorts for account and container ids, so there is no confusion.